Form-Controls/index.html

```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label> Name
+      <input required type="text" placeholder="Bereket Hayelom">
+      </label>
+  </div>
+  <br>
+  <div> 
+      <label> Email
+      <input required type="email" placeholder="bereket@example.com">
+      </label>
+  </div>
+  <br>
+  <div> 
```

Comment:
How can you ensure the user picks both a color and size before ordering?

Reply:
i add the required on my html to make sure they submitted both